src/views/vmd-rdv.view.ts

```diff
@@ -404,6 +420,40 @@ export abstract class AbstractVmdRdvView extends LitElement {
             throw new Error(`Unsupported tri : ${tri}`);
         }
     }
+    
+    protected filterTypeVaccin(vaccine_type: Array, typeVaccin: CodeTypeVaccin) {
+        if(typeVaccin === 'tous') {
+            return true;
+        } else {
+            let result = false;
+            
+            if(vaccine_type && vaccine_type.length)
+            {
+                if(vaccine_type.indexOf(',')!==-1)
+                {
+                    let arrayVaccinne = vaccine_type.split(',');
```

Comment:
`const ` est plus approprié
```suggestion
                    const arrayVaccinne = vaccine_type.split(',');
```